app/views/content/non-uk-teachers/non-uk-qualifications.md

```diff
@@ -62,73 +58,61 @@ keywords:
 
 ## If you do not have a bachelor's degree
 
-If you do not have a degree, you can apply for an [undergraduate course in England on the UCAS website](https://www.ucas.com/postgraduate/teacher-training/applying-teacher-training/find-teacher-training-programmes). These undergraduate courses are 3 to 4 years and combine a teaching degree with [qualified teacher status (QTS)](/train-to-be-a-teacher/what-is-qts). When you complete your studies, you’ll be able to apply for jobs as a teacher in England.
+If you do not have a degree, you can apply for an [undergraduate course in England on the UCAS website](https://www.ucas.com/postgraduate/teacher-training/applying-teacher-training/find-teacher-training-programmes). These undergraduate courses are 3 to 4 years and you can search for courses that combine a teaching degree with [qualified teacher status (QTS)](/train-to-be-a-teacher/what-is-qts). When you complete your studies, you’ll be able to apply for jobs as a teacher in England.
 
 The [UK Council for International Student Affairs website](https://www.ukcisa.org.uk/) has lots of helpful information about studying at an English university.
 
-## How to apply
+## How to apply for teacher training
+
+You can [search for postgraduate teacher training courses](https://find-teacher-training-courses.service.gov.uk/) that lead to qualified teacher status (QTS).  
+
+[Find guidance on applying for teacher training courses](/how-to-apply-for-teacher-training/teacher-training-application). 
 
-Have you spoken to one of our advisers to check you have the required qualifications for teacher training in England? They can also help you submit your application.
+When you get offered a place on a course, your provider will arrange a background and criminal records check. This is known as a Disclosure and Barring Service (DBS) check. You must pass these checks to teach students under 18. Safeguarding checks may take longer if you are living or have lived abroad. 
 
-Learn more about [how to find and apply for teacher training courses](/non-uk-teachers/train-to-teach-in-england-as-an-international-student#find-and-apply-for-teacher-training-courses).
+You’ll also need to complete a fitness questionnaire before starting training. 
 
-## Already a qualified teacher?
+Due to rules about immigration, not all providers can accept international applicants. Check with providers before you apply.
 
 If you're already a qualified teacher, find out how you can [teach in England if you completed your teacher training outside the UK](/non-uk-teachers/teach-in-england-if-you-trained-overseas).
 
-## Contact
+## Get help
 
-If you have general questions about the information on this page, you can email us at teach.inengland@education.gov.uk.
+To get help with your qualifications, you can use the live chat service at the bottom of this page. Or you can call us on +44 800 389 2500. Calls will be charged at your country's standard rate.
```

Comment:
I think this sounds like we are going to help people get their qualifications - should this be more about `get help with checking your qualifications`?